src/processor/negativeDataFilter.ts

```diff
@@ -0,0 +1,44 @@
+import { StageHandler } from '../util/types';
```

Comment:
This should appear after the license header.

Reply:
fixed